test/helpers/cluster_resource_discovery_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module ClusterResourceDiscoveryHelper
+  def mocked_cluster_resource_discovery(response, success: true)
+    Krane::Kubectl.any_instance.stubs(:run)
+      .with("api-resources", "--namespaced=false", attempts: 5, use_namespace: false, output: "wide")
+      .returns([response, "", stub(success?: success)])
+    Krane::ClusterResourceDiscovery.new(task_config: task_config, namespace_tags: [])
+  end
+
+  def api_versions_full_response
```

Comment:
Seems odd to have this in here but not something to mock its kubectl call

Reply:
Straight copy pasta, I can iron this out better for sure. Will do.